setup.py

```diff
@@ -29,7 +29,7 @@
     'openapi-typed_2>=0.0.2',
     'typeguard>=2.7.0',
     'genson',
-    'http-types==0.0.11',
+    'http-types==0.0.13',
```

Comment:
I would be brave here and set `http-types>=0.0.13,<0.1.0` and try and bump the version when adding breaking changes from now on.

Reply:
Makes sense, did that now!